assets/js/util/sanitize.test.js

```diff
@@ -29,3 +29,19 @@ describe( 'sanitizeHTML', () => {
 		} );
 	} );
 } );
+
+describe( 'unTrailingSlashIt', () => {
+	it( 'should not change a URL without trailing slash', () => {
+		const url = 'http://example.org';
+		expect( unTrailingSlashIt( url ) ).toEqual( 'http://example.org' );
+	} );
+
+	it( 'should change a URL with trailing slash', () => {
+		const url = 'http://example.org/';
+		expect( unTrailingSlashIt( url ) ).toEqual( 'http://example.org' );
+	} );
+
+	it( 'should return null if the parameter is not a string', () => {
+		expect( unTrailingSlashIt( 1 ) ).toBeNull();
+	} );
+} );
```

Comment:
Let's add a case or two here to show that it removes more than one consecutive slash at the end if present. E.g. `http://` -> `http:`